lib/rucio/db/sqla/types.py

```diff
@@ -113,5 +113,7 @@ def load_dialect_impl(self, dialect):
             return dialect.type_descriptor(JSONB())
         elif dialect.name == 'mysql':
             return dialect.type_descriptor(types.JSON())
+        elif dialect.name == 'oracle':
+        	return dialect.type_descriptor(CLOB())
```

Comment:
lib/rucio/db/sqla/types.py:117:9: E101 indentation contains mixed spaces and tabs
lib/rucio/db/sqla/types.py:117:9: W191 indentation contains tabs